code/game/machinery/cloning.dm

```diff
@@ -299,20 +304,20 @@
 		biomass += BIOMASS_MEAT_AMOUNT
 		show_message = 1
 	if(show_message)
-		visible_message("\The [src] sucks in and processes the nearby biomass.")
+		visible_message("[src] sucks in and processes the nearby biomass.")
 
 	if(stat & NOPOWER) //Autoeject if power is lost
 		if(occupant)
 			locked = 0
 			go_out()
-		return
+			connected_message("Clone Ejected: Loss of power.")
 
-	if((occupant) && (occupant.loc == src))
-		if((occupant.stat == DEAD) || (occupant.suiciding) || !occupant.key)  //Autoeject corpses and suiciding dudes.
+	else if((occupant) && (occupant.loc == src))
+		if((occupant.stat == DEAD) || (occupant.suiciding))  //Autoeject corpses and suiciding dudes.
 			locked = 0
 			go_out()
 			connected_message("Clone Rejected: Deceased.")
-			return
+			Radio.autosay("The cloning of <b>[occupant]</b> has been aborted due to unrecoverable tissue failure.", name, "Medical", list(z))

```

Comment:
By the time this message occurs, `go_out` will have nulled out `occupant`. Would be fine if it was called *after* these messages, like it is for successful clones.